roles/installer/templates/deployment.yaml.j2

```diff
@@ -222,6 +222,10 @@ spec:
               mountPath: "/var/run/awx-rsyslog"
             - name: rsyslog-dir
               mountPath: "/var/lib/awx/rsyslog"
+            - name: "{{ meta.name }}-receptor-config"
+              mountPath: "/etc/receptor/receptor.conf"
+              subPath: receptor.conf
+              readOnly: true
```

Comment:
@rooftopcellist  I have not tested yet (I'll test it tomorrow), but `awx` code reads the `/etc/receptor/receptor.conf` to find the `socket` location to dispatch work (see https://github.com/ansible/awx/pull/10932) . 

Looking at the `configmap`, the socket will be available at `filename: /var/run/receptor/receptor.sock`  with perms `0660` 
```yaml
- control-service:
    service: control
    filename: /var/run/receptor/receptor.sock
    permissions: 0660
```